### PR TITLE
[fixes #359] removed util.puts/util.print to remove warnings

### LINKED ISSUE #359: util.print is deprecated
For the following files there are `util.prints` statements. Can we make them `console.log` as they are deprecated?
 

  * vows/lib/vows.js
  * vows/lib/vows/reporters/json-file.js

Thanks!

### PR DESCRIPTION
**Pull request by edwardhotchkiss**

### CODE REVIEW

File: lib/vows.js [hunk 1]
```diff
@@ -206,7 +206,9 @@ process.on('exit', function () {
                 }
             });
 
-            if (unFired.length > 0 || expected.length > 0) { util.print('\n') }
+            if (unFired.length > 0 || expected.length > 0) {
+                console.log('\n')
```
Comment (indexzero) [inline lib/vows.js]: I think due to legacy reasons this may need to be `global.console.log`

File: lib/vows.js [hunk 2]
```diff
@@ -235,7 +237,7 @@ process.on('exit', function () {
         });
     });
     if (failure) {
-        util.puts(console.result(results));
+        console.result(results);
```
Comment (indexzero) [inline lib/vows.js]: `global.console.log(console.result(results));`

File: lib/vows/reporters/json-file.js [hunk 3]
```diff
@@ -38,7 +38,7 @@ this.report = function (obj) {
             fs.close(out);
             puts("Generating report.json file complete")
         } catch (error) {
-            util.print("Error: Unable to write to file report.json\n");
+            console.error("Error: Unable to write to file report.json\n");
```
Comment (indexzero) [inline lib/vows/reporters/json-file.js]: `global.console.error`

**Comment by indexzero** [comment]
`vows` defines it's own `console` (although not globally) so you just need to explicitly hook into the `global.console` to change scopes.

**Comment by edwardhotchkiss** [comment]
Fixed, tests passing.

**Comment by indexzero** [comment]
Meged into `0.9.0-rc3`

**Pull request merged**